src/components/pages/Leaderboard/RenderLeaderboard.js

```diff
@@ -51,27 +75,44 @@ for (var i = 0; i < sortedDummyData.length; i++) {
   sortedDummyData[i].id = i + 1;
 }
 
-const TabPane = Tabs;
+const sortedDummyImprovedData = mergeSort(dummyImprovedData).reverse();
+
+for (var k = 0; k < sortedDummyImprovedData.length; k++) {
+  sortedDummyImprovedData[k].id = k + 1;
+}
 
 function RenderLeaderboard(props) {
   return (
     <LayoutContainer>
       <NavBar titleName={'Leaderboard'} backgroundColor="#293845" />
-      <ul>
-        {sortedDummyData.map(elem => (
-          <div className={`li-container`}>
-            <li>
-              <h2 className="place">{elem.id}</h2>
-              <h2 className="place">{elem.clubname}</h2>
-              <h2 className="rating">{elem.rating}</h2>
-            </li>
-          </div>
-        ))}
-        <Tabs defaultActiveKey="1">
-          <TabPane tab="Highest Sentiment Last Week" key="1"></TabPane>
-          <TabPane tab="Most Improved Since Last Mon" key="2"></TabPane>
-        </Tabs>
-      </ul>
+      <Tabs>
+        <div label="HIGHEST SENTIMENT LAST WEEK">
+          <ul>
+            {sortedDummyData.map(elem => (
+              <div className={`li-container`}>
+                <li>
+                  <h2 className="place">{elem.id}</h2>
+                  <h2 className="place">{elem.clubname}</h2>
+                  <h2 className="rating">{elem.rating}</h2>
+                </li>
+              </div>
+            ))}
+          </ul>
+        </div>
+        <div label="MOST IMPROVED LAST MONTH">
+          <ul>
+            {sortedDummyImprovedData.map(elem => (
```

Comment:
mapped react elements should have a key could just be key={elem.id}

Reply:
Gotchu!